fix(learn): Remove arrows from test name assertion

naomi-lgbt opened the pull request:
Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->

As noted in https://github.com/freeCodeCamp/freeCodeCamp/pull/40358, @moT01 caught an issue with the way we are differentiating between unit/functional tests in a camper's project. Because we've removed the test structure from the boilerplate, the format for the "test context" can differ if the user nests the test declarations on a different level. This changes the placement of the arrow symbol in the context name. 

For better future-proofing, I removed the arrow from the assertion entirely. I fixed the Metric-Imperial Converter in the existing (linked) PR, so this PR is to catch the other projects. From what I could see, only the American British Translator had the arrow present.

Commit:
fix: Remove arrows from test name assertion

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

ofk8vb reviewed (dismissed):
I was stuck in the project due to issue discussed here. This change would have fixed it.

ShaunSHamilton reviewed (commented):
Just to remind: It would be beneficial to include on line `104` (in the time user story), something along the lines of:
```
The whole time should be wrapped within the `span` element.
```

That is, to clarify not just the `.` or `:` (`12<span highlight="class">:</span>15`)

Commit:
fix: Add note about time span

Adds a note regarding the expectation of the `span` element to wrap the
entire time string when translated, not just the delimiter symbol.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

ShaunSHamilton reviewed (dismissed):
This LGTM 👍 
Thank you, Nicholas.

RandellDawson commented on curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/american-british-translator.md:
````diff
@@ -101,7 +101,7 @@ async (getUserInput) => {
 ```
 
-The `/api/translate` route should handle the way time is written in American and British English. For example, ten thirty is written as "10.30" in British English and "10:30" in American English.
+The `/api/translate` route should handle the way time is written in American and British English. For example, ten thirty is written as "10.30" in British English and "10:30" in American English. NOTE: The `span` element should wrap the full time string, i.e. `<span class="highlight">10:30</span>`.
````
@nhcarrigan Though not specified in the [docs](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-work-on-coding-challenges.md), notes should be only a separate line.  The documentation does specify the format of the note (see below).  I also am suggesting to change the word "full" to "entire".

```suggestion
The `/api/translate` route should handle the way time is written in American and British English. For example, ten thirty is written as "10.30" in British English and "10:30" in American English.

**Note:** The `span` element should wrap the entire time string, i.e. `<span class="highlight">10:30</span>`.
```

ShaunSHamilton replied:
I think we could add the mention of a new line, because this is mentioned:
> - Additional information in the form of a note should be formatted `**Note:** Rest of note text...`

https://github.com/freeCodeCamp/freeCodeCamp/blame/master/docs/how-to-work-on-coding-challenges.md#L279

RandellDawson reviewed (changes requested):
I have a suggested change.

Commit:
fix: use correct note syntax

Co-authored-by: Randell Dawson <5313213+RandellDawson@users.noreply.github.com>

nhcarrigan commented:
@RandellDawson in this case, because it's a test text, the new line is causing the build to fail.

Commit:
fix: remove note syntax

New line was breaking the build, so changing the `note` comment to be
regular text.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

RandellDawson commented:
> in this case, because it's a test text, the new line is causing the build to fail.

Makes sense.

RandellDawson commented:
@ShaunSHamilton Are you fine with these changes?

ShaunSHamilton reviewed (approved):
This still looks good. Thanks, Nicholas 👍

RandellDawson commented:
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉
